packages/admin-panel-server/src/routes/UserRoute.ts

```diff
@@ -6,15 +6,23 @@
 import camelcaseKeys from 'camelcase-keys';
 import { Route } from '@tupaia/server-boilerplate';
 import { AccessPolicy } from '@tupaia/access-policy';
-import { BES_ADMIN_PERMISSION_GROUP } from '../constants';
+import { BES_ADMIN_PERMISSION_GROUP, VIZ_BUILDER_USER_PERMISSION_GROUP } from '../constants';
 
 const isBESAdmin = (policy: Record<string, string[]>) => {
   return new AccessPolicy(policy).allowsSome(undefined, BES_ADMIN_PERMISSION_GROUP);
 };
 
+const isVizBuilderUser = (policy: Record<string, string[]>) => {
+  return new AccessPolicy(policy).allowsSome(undefined, VIZ_BUILDER_USER_PERMISSION_GROUP);
+};
+
 export class UserRoute extends Route {
   public async buildResponse() {
     const user = await this.req.ctx.services.central.getUser();
-    return { ...camelcaseKeys(user), isBESAdmin: isBESAdmin(user.accessPolicy) };
+    return {
+      ...camelcaseKeys(user),
+      isBESAdmin: isBESAdmin(user.accessPolicy),
+      isVizBuilderUser: isVizBuilderUser(user.accessPolicy),
```

Comment:
```suggestion
      isBESAdmin: isBESAdmin(user.accessPolicy),
      isVizBuilderUser: isVizBuilderUser(user.accessPolicy),
```
I know this pattern pre-dates your change, but can we get away with not adding these keys on the backend and just working them out in the front-end? It feels quite ugly for an endpoint to be passing around derivable data like this.